### PR TITLE
Use custom events for shed

### PR DESCRIPTION
**Pull request by JohnMcGuinness**
Updated collapsibleToggle to use custom events for SHED.

@ricksbrown is this okay?

### CODE REVIEW

File: wcomponents-theme/src/main/js/wc/ui/tabset.js [hunk 1]
```diff
@@ -464,32 +464,33 @@ function(toArray, ariaAnalog, formUpdateManager, getFilteredGroup, initialise, s
...
 							break;
 						default:
 							console.warn("Unknown action", action);
 							break;
 					}
-				} else if ((action === shed.actions.DISABLE || action === shed.actions.ENABLE) && TABLIST.isOneOfMe(element)) {
+				} else if ((action === shed.events.DISABLE || action === shed.events.ENABLE) && TABLIST.isOneOfMe(element)) {
 					// if the tablist is disabled or enabled, diable/enable all the tabs.
 					Array.prototype.forEach.call(this.ITEM.findDescendants(element), function (next) {
 						shed[action](next);
```
Comment (JohnMcGuinness) [inline wcomponents-theme/src/main/js/wc/ui/tabset.js]: Would the above line ```shed[action](next)``` be OK. Although no tests are failing, I think that would end up being something like ```shed['wcdisable'](next)```. There's no ```wcdisable``` method on shed, so should it be mapped back to ```disable``` or ```enable```.

**Comment by JohnMcGuinness** [comment]
@ricksbrown I've made all of the initial changes that I think need to be made. I'm unsure if there should be new tests, but I'm sure you'll let me know once you review.

File: wcomponents-theme/src/main/js/wc/ui/tabset.js (hunk 1 above)
Comment (JohnMcGuinness) [reply wcomponents-theme/src/main/js/wc/ui/tabset.js]: I think ```shed.events[action]``` would map it back.

Comment (ricksbrown) [reply wcomponents-theme/src/main/js/wc/ui/tabset.js]: Yes this looks like it won't work as is - mapping it back would be an easy solution

**Pull request closed without merging**